hello-world/hello-world-io/src/lib.rs

```diff
@@ -2,15 +2,15 @@
 
 use codec::{Decode, Encode};
 use gmeta::{InOut, Metadata};
-use scale_info::TypeInfo;
 use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
 
 pub struct ProgramMetadata;
 
 impl Metadata for ProgramMetadata {
-   type Init = InOut<String,()>;
-   type Reply = ();
-   type Others = ();
+   type Init = InOut<String, ()>;
+   type Reply = InOut<(), ()>;
+   type Others = InOut<(), ()>;
```

Comment:
It will be fixed in academy materials, so for your information: it is possible to use `gmeta::In` here, and no need to point unit input/output.
```suggestion
   type Init = In<String>;
   type Reply = ();
   type Others = ();
```